clusterloader2/testing/load/config.yaml

```diff
@@ -366,6 +392,147 @@ steps:
       action: gather
   {{end}}
 
+# BEGIN scheduler throughput
+- name: Creating scheduler throughput measurements
+  measurements:
+  - Identifier: HighThroughputStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = scheduler-throughput
+      threshold: 1h # TODO(https://github.com/kubernetes/perf-tests/issues/1024): Ideally, this should be 5s
+  - Identifier: WaitForSchedulerThroughputDeployments
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: apps/v1
+      kind: Deployment
+      labelSelector: group = scheduler-throughput
+      # The operation timeout shouldn't be less than 20m to make sure that ~10m node
+      # failure won't fail the test. See https://github.com/kubernetes/kubernetes/issues/73461#issuecomment-467338711
+      operationTimeout: 20m
+  - Identifier: SchedulingThroughput
+    Method: SchedulingThroughput
+    Params:
+      action: start
+      labelSelector: group = scheduler-throughput
+- name: Creating scheduler throughput pods
+  phases:
+  - namespaceRange:
+      min: {{AddInt $namespaces 1}}
+      max: {{AddInt $namespaces $schedulerThroughputNamespaces}}
+    replicasPerNamespace: 1
+    tuningSet: PodThroughputParallel
```

Comment:
PodThroughputParallel name is misleading, as in fact we create deployments with that throughput. Given those are fairly big ones, pod throughput in fact may potentially be much higher.

I don't have a good name though...

Reply:
Doh, I wanted to write SchedulerThroughputParallel, meaning that this is a dedicated tuningSet for SchedulerThroughput that results in fully parallel creations of deployments. 
Changed and documented better.